src/cli/template_test.go

```diff
@@ -65,6 +66,7 @@ func TestShouldRenderDockerTemplate(t *testing.T) {
 				Name:      string(projectType),
 				Directory: path.Join(root, props["directory"]),
 				Resources: os.DirFS(root),
+				Language:  "auto",
```

Comment:
I'm conflicted about this, not sure "auto" is needed, could we do something like app-name where we detect it and if we cannot with make it mandatory?